covid19-notebooks/requirements.txt

```diff
@@ -1,10 +1,10 @@
 fastai==1.0.61
 gen3
-matplotlib==3.4.3
-numpy==1.21.4
-pandas==1.3.4
-plotly==5.5.0
+matplotlib
```

Comment:
why did you remove the pinned versions?
it's fine but we'll need to test the new [jupyter-covid19 image](https://github.com/uc-cdis/containers/blob/master/jupyter-covid19/Dockerfile) and see if the notebooks still work

Reply:
I didn't touch this file. If you compare the [master branch](https://github.com/uc-cdis/covid19-tools/blob/master/covid19-notebooks/requirements.txt) and [my branch](https://github.com/uc-cdis/covid19-tools/blob/feat/sir-bayes/covid19-notebooks/requirements.txt), they are the same. Don't know why GitHub shows the change.

Reply:
oh then could you rebase your branch on master

Reply:
I rebased my branch on top of master, and this change disappeared